cmd/apply.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"gopkg.in/nullstone-io/nullstone.v0/runs"
+	"os"
+	"strings"
+)
+
+var Apply = func() *cli.Command {
+	return &cli.Command{
+		Name:      "Apply",
+		Usage:     "Runs an apply with optional auto-approval",
+		UsageText: "nullstone plan [--stack=<stack-name>] --block=<block-name> --env=<env-name> [options]",
```

Comment:
Should this usage text say `nullstone plan`? Shouldn't it be `nullstone apply`?

Reply:
Done.